src/test/java/components/ui/BaseTest.java

```diff
@@ -1,64 +1,67 @@
 package components.ui;
 
+import com.ui.BasePage;
 import io.github.bonigarcia.wdm.WebDriverManager;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
 import org.openqa.selenium.edge.EdgeDriver;
 import org.openqa.selenium.edge.EdgeOptions;
-import org.openqa.selenium.firefox.FirefoxDriver;
-import org.openqa.selenium.ie.InternetExplorerDriver;
 import org.openqa.selenium.remote.CapabilityType;
 import org.openqa.selenium.remote.RemoteWebDriver;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.slf4j.event.EventRecodingLogger;
-import utilities.readPropertyFile;
+import utilities.ReadPropertyFile;
 
+import java.io.File;
 import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
 import static org.openqa.selenium.remote.CapabilityType.PROXY;
 
-public class BaseTest {
-    public static WebDriver driver;
-    readPropertyFile fr = new readPropertyFile();
-    private final static Logger logger = LoggerFactory.getLogger(BaseTest.class);
+public class BaseTest  {
+    ReadPropertyFile fr = new ReadPropertyFile();
+    private WebDriver driver;
 
-    public WebDriver getDriver() throws IOException {
-        String browser="";
-      if (driver==null)
-      {
-           browser =  fr.getPropertyValue("browser");
-      }
-      if (browser.equalsIgnoreCase("edge"))
+
+    public WebDriver setUp() throws IOException {
+
+        String browser =  fr.getPropertyValue("browser");
+
+        Path resourceDirectory = Paths.get("src","test","resources");
+        String absolutePath = resourceDirectory.toFile().getAbsolutePath();
+
+        if (browser.equalsIgnoreCase("edge"))
```

Comment:
make refactoring of this method, split this method up for few methods or remove one of the drivers, lets say edge and use only Chrome (as far as I remember we are not going to run cross browser testing, one driver setUP should be enough for now). I can see some duplicates of code, like:
1. String[] pathNames = {absolutePath, "\\drivers\\msedgedriver.exe" };
                String path = String.join(File.pathSeparator, pathNames);
                System.setProperty("webdriver.edge.driver",path.replaceAll(";","")); 
the difference is driver name only, it can be moved to independent method like setSystemProperty(String browserName)

2.                 options.setCapability(CapabilityType.ACCEPT_INSECURE_CERTS, true);
                options.addArguments("--remote-allow-origins=*");
                options.setCapability("ignore-certificate-errors", true);
all those setters are equals for both browsers,

Better to clean the code from duplicates.